editor/scene/gui/control_editor_plugin.cpp

```diff
@@ -596,8 +596,32 @@ void ControlEditorPresetPicker::_add_separator(BoxContainer *p_box, Separator *p
 	p_box->add_child(p_separator);
 }
 
+void ControlEditorPresetPicker::_update_preset_button_state(int p_preset) {
+	for (KeyValue<int, Button *> &E : preset_buttons) {
+		Button *button = E.value;
+
+		if (!button) {
+			continue;
+		}
+
+		button->begin_bulk_theme_override();
+
+		if (E.key == p_preset) {
+			const Color pressed_color = get_theme_color("icon_pressed_color", "Button");
```

Comment:
Inconsistent use of SNAME
```suggestion
			const Color pressed_color = get_theme_color(SNAME("icon_pressed_color"), "Button");
```
Although any reason why it's not a toggleable button?